### PR TITLE
Fix the metric name in `KerasPruningCallback` example.

### PR DESCRIPTION
**Pull request by toshihikoyanase**
## Motivation

`examples/pruning/keras_integration.py` does not prune any trials. This is because the metric name has been changed from `val_acc` to `val_accuracy`.

## Changes

This PR simply updates the metric name, but it may be better to show warning messages if the metric cannot be found in the callback. If you have any comments, please let me know.

### CODE REVIEW

**Comment by keisuke-umezawa** [comment]
FYI
It seems that we can run the example if we write `metrics=["acc"],` here.

https://github.com/optuna/optuna/blob/b5f6a262084f42a01054d80bf91dbf76e60c9258/examples/pruning/keras_integration.py#L49

**Comment by keisuke-umezawa** [comment]
Sorry, just a nit.
If possible, could you update this part from ``val_acc`` to ``val_accuracy``?

https://github.com/optuna/optuna/blob/master/optuna/integration/keras.py#L32

**Comment by toshihikoyanase** [comment]
> If possible, could you update this part from val_acc to val_accuracy?
> https://github.com/optuna/optuna/blob/master/optuna/integration/keras.py#L32

@keisuke-umezawa Good catch! I fixed it in https://github.com/optuna/optuna/pull/1218/commits/ddb4cc3bf224489858f183530bf1002078277bc3. Thank you.
